python_test/test_code_util.py

```diff
@@ -0,0 +1,81 @@
+from ldpc.codes import hamming_code, rep_code, ring_code, random_binary_code
+import scipy.sparse
+import numpy as np
+from ldpc import mod2
+from ldpc import code_util
+
+def test_construct_generator_matrix():
+
+    for i in range(3,10):
+
+        H = hamming_code(i)
```

Comment:
I guess using pytest.fixture would be the best practice to generate test objects used (also those used across different tests)